apps/web/src/server/applications/case/representations/representation-details/application-representation-details.view-model.js

```diff
@@ -42,37 +43,27 @@ import { format } from 'date-fns';
 /**
  * @param {object} representation
  * @param {Contact[]} representation.contacts
- * @param {string} type
- * @returns {object}
+ * @returns {object[]}
  */
-const getContactDetailsByContactType = ({ contacts }, type) => {
-	let result = {};
-
-	for (const contact of contacts) {
-		const contactType = contact.type.toLowerCase();
-
-		if (contactType === type) {
-			result = {
-				orgName: contact.organisationName || '',
-				name: `${contact.firstName} ${contact.lastName}`,
-				orgOrName: contact.organisationName
-					? contact.organisationName
-					: `${contact.firstName} ${contact.lastName}`,
-				jobTitle: contact.jobTitle || '',
-				under18: contact.under18 ? 'Yes' : 'No',
-				email: contact.email || '',
-				phoneNumber: contact.phoneNumber || '',
-				preferredContact: '',
-				addressLine1: contact.address.addressLine1 || '',
-				addressLine2: contact.address.addressLine2 || '',
-				town: contact.address.town || '',
-				county: contact.address.county || '',
-				postcode: contact.address.postcode || ''
-			};
-		}
-	}
-
-	return result;
+const getContactDetailsByContactType = ({ contacts }) => {
+	return contacts.map((contact) => ({
+		type: contact.type,
+		orgName: contact.organisationName || '',
+		name: `${contact.firstName} ${contact.lastName}`,
+		orgOrName: contact.organisationName
+			? contact.organisationName
+			: `${contact.firstName} ${contact.lastName}`,
+		jobTitle: contact.jobTitle || '',
+		under18: contact.under18 ? 'Yes' : 'No',
```

Comment:
does this need to allow for `contact.under18` possibly being `null` / unknown?

Reply:
Good spot need to check the details page ticket.

Reply:
we have added a func to handle unknown logic state. 

Other screens tickets have been updated / considered.